src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -149,9 +166,12 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
       stay()
     case Event(ExecutionStoreCreated(startMode), NoFailureMessage) =>
       if (startMode == Restart) {
-        symbolsMarkdownTable foreach { table => log.info(s"Initial symbols:\n\n$table") }
-        executionsMarkdownTable foreach { table => log.info(s"Initial executions:\n\n$table") } 
+        dumpTables()
+        stay()
+      } else {
+        startRunnableCalls()
       }
+    case Event(TablesDumped, NoFailureMessage) =>
       startRunnableCalls()
```

Comment:
Ok, I guess I just find it strange to have a DumpTables Actor Event, because it's just logging Database Tables and is not really related to what this Actor does. IMO I would say it's better, if possible, to keep Actor Events for things that have "business meaning" to what the actor does (in this case handling Workflow execution), while also limiting the amount of messages in the actor queue.
Maybe making dumpTables return a `Future[Unit]` and moving the `onComplete` part on line 169 would remove the need for the DumpTables Event.
But that's far from being super critical, it's more of a general comment, feel free to keep it the way it is if you think it's better.